ecommers/lib/core/services/extensions/get_it_extension.dart

```diff
@@ -1,22 +1,44 @@
+import 'dart:io';
+
 import 'package:chopper/chopper.dart';
-import 'package:ecommers/core/common/json_serializable_converter.dart';
-import 'package:ecommers/core/models/index.dart';
-import 'package:ecommers/core/models/login_model.dart';
-import 'package:ecommers/core/services/api_service.dart';
-import 'package:ecommers/web_server/local_server.dart';
 import 'package:get_it/get_it.dart';
 
+import '../../../web_server/local_server.dart';
+import '../../common/index.dart';
+import '../../models/data_models/index.dart';
```

Comment:
use full path

Reply:
according to style guides, we shouldn't use full path